### PR TITLE
fix: gaiad export to stdout, not stderr

### PR DESCRIPTION
**Pull request by yaruwangway**
Please check https://github.com/cosmos/gaia/issues/2249 for issue description.
 - [x] fix `gaiad export` to output stdout
 - [x] fix parsing block info from stdout, not stderr 
 - [ ] add to changelog (need to decide which version to include it)

### CODE REVIEW

**Comment by yaruwangway** [comment]
Need to decide: which version to include this fix.

File: cmd/gaiad/cmd/root.go [hunk 1]
```diff
@@ -52,6 +52,10 @@ func NewRootCmd() (*cobra.Command, params.EncodingConfig) {
 		Use:   "gaiad",
 		Short: "Stargate Cosmos Hub App",
 		PersistentPreRunE: func(cmd *cobra.Command, _ []string) error {
+			// set the default command outputs
+			cmd.SetOut(cmd.OutOrStdout())
+			cmd.SetErr(cmd.ErrOrStderr())
+
```
Comment (mpoke) [inline cmd/gaiad/cmd/root.go:58]: What's the impact of this change on users that already use stderr?

**Review by mpoke — CHANGES_REQUESTED** [review]
Not sure whether we want to fix this in a patch release.

File: cmd/gaiad/cmd/root.go (hunk 1 above)
Comment (yaruwangway) [reply cmd/gaiad/cmd/root.go:58]: if users use `gaiad export > genesis.json 2>&1` before, it will still work.

**Comment by mpoke** [comment]
Closing due to lack of activity.

**Pull request closed without merging**